### PR TITLE
Final implementation

### PR DESCRIPTION
**Pull request by HHjolany**
## Link to Click Up ticket

[ticket (12.4) Api](https://app.clickup.com/t/1da3d6p)
## Describe this PR
due to large file changes this PR is one of the future series PR
### *What is the problem we're trying to solve*

this pull request for final implementation 

### *What changes have we introduced*

these changes consist of final implementation

#### _Checklist_

- [x] Finalize

### CODE REVIEW

File: AccountsApi/appsettings.json [hunk 1]
```diff
@@ -6,13 +6,8 @@
       "Microsoft": "Information"
     }
   },
-  "XRay": {
```
Comment (LBHSPreston) [inline AccountsApi/appsettings.json:9]: This shouldn't be removed.  We need this in our APIs for monitoring and debugging.

File: AccountsApi/appsettings.json [hunk 2]
```diff
@@ -6,13 +6,8 @@
...
-    "DisableXRayTracing": "false",
-    "UseRuntimeErrors": "true",
-    "CollectSqlQueries": "false"
-  },
   "AWS": {
-    "Region": "eu-west-2"
+    "Region": "DefaultRegion"
```
Comment (LBHSPreston) [inline AccountsApi/appsettings.json]: The correct region should be supplied here.

**Review by LBHSPreston — CHANGES_REQUESTED** [review]

File: AccountsApi/appsettings.json (hunk 2 above)
Comment (HHjolany) [reply AccountsApi/appsettings.json]: sorry / did
thanks

File: AccountsApi/appsettings.json (hunk 1 above)
Comment (HHjolany) [reply AccountsApi/appsettings.json]: done mate.

**Review by LBHSPreston — APPROVED** [review]

**Pull request merged**